contributor-guide/modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,10 @@
-* New Content
-* xref:index.adoc[Contributor Guide]
+
+
 * xref:intro.adoc[Introduction to becoming a Boost Contributor]
+** xref:license-requirements.adoc[License Requirements]
```

Comment:
Please use navtitle.

Reply:
navtitles added to all topics in the Contributor guide